lib/apress/selenium_products/version.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Apress
+  module SeleniumProducts
+    VERSION = '1.0.0'.freeze
```

Comment:
[Correctable] Style/RedundantFreeze: Do not freeze immutable objects, as freezing them has no effect.